gramps/gen/db/__init__.py

```diff
@@ -71,3 +71,5 @@
 from .undoredo import *
 from .utils import *
 from .generic import *
+
+Database = DbGeneric
```

Comment:
Is it clearer to use `DbGeneric` rather than introduce a new type alias `Database`? (at least I think that's what you are doing)
Even if I find `Database` easier to read, I'm not sure having two names for the same type helps.

Reply:
Yeah, I agree. But I really didn't want to use the `DbGeneric` name when that doesn't convey the right information to the humans. Another item we can wrestle with as we come up with better type infrastructure.

Reply:
I'll try to define a Database type...